src/components/Quests/QuestPage/BackButton/BackButton.tsx

```diff
@@ -35,7 +35,9 @@ export const BackButton = ({ path, title }: BackButtonProps) => {
           fontWeight={500}
           marginLeft={'8px'}
         >
-          {String(title || t('navbar.navbarMenu.profile')).toUpperCase()}
+          {String(
```

Comment:
String casting should not be needed

Reply:
Oh! I copied that from a different file. I suppose we can also remove it over there. Will do!